app/services/migrate_paperclip_data.rb

```diff
@@ -6,17 +6,17 @@ class MigratePaperclipData
   attr_accessor :new_attachment_name
   attr_accessor :model_class
 
-  def self.perform(model_class, attachments_to_process)
+  def self.perform(model_class, attachment_names)
```

Comment:
switched these so they derive the new attachment names, should make passing args to the rake task more trivial